docs/issuer-service.yml

```diff
@@ -36,6 +36,25 @@ paths:
           description: invalid input!
         "500":
           description: error!
+
+  /credentials/{id}:
```

Comment:
Is this the right name for it? Shouldn't it explicitly indicate revocation?

Reply:
see our current api for why it is the way it is here.

Reply:
We could of course change our API. 

But I would prefer to only do that if we need to based on what the spec here says.